templates/prescription/general_edit.html

```diff
@@ -272,6 +271,20 @@
 <!-- for the fancy jQuery stuff -->
 <script type="text/javascript">
 
+
+    $('.datepicker').datetimepicker({
+{/literal}
+    {php}
+    $datetimepicker_timepicker = false;
+    $datetimepicker_showseconds = false;
+    $datetimepicker_formatInput = true;
+    require($GLOBALS['srcdir'] . '/js/xl/jquery-datetimepicker-2-5-4.js.php');
+    {/php}
```

Comment:
We can no longer use php within smarty templates (note we converted all scripts to this rule except for the calendar) in order to work towards smart3 migration in the future. Recommend checking out another smarty script that includes the datepicker to see how to do it (the patient documents section uses a datepicker within smarty).

Reply:
Fixed..
Very nice function..